ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ReadOnlyServiceCallAuthTests.scala

```diff
@@ -25,25 +25,25 @@ trait ReadOnlyServiceCallAuthTests extends ServiceCallWithMainActorAuthTests {
       userPrefix: String,
       rights: Vector[proto.Right.Kind],
   ): Future[Any] =
-    createUserByAdmin(userPrefix + mainActor, rights.map(proto.Right(_)))
-      .flatMap { case (_, token) => serviceCallWithoutApplicationId(token) }
+    createUserByAdmin(userPrefix + mainActor, rights = rights.map(proto.Right(_)))
+      .flatMap { case (_, token) => serviceCall(ServiceCallContext(token)) }
```

Comment:
includeApplicationId = false?

Reply:
no, it wont work, this was a hack I removed, `serviceCallWithoutApplicationId` was a different impl from `serviceCallWithToken`